docs/how_tos/i18n.rst

```diff
@@ -184,8 +184,26 @@ Create your pipeline job on Jenkins
 
      * Note that the Jenkins machine uses UTC.
 
+#. Test your change locally.
+
+   * Clone https://github.com/openedx/ecommerce-scripts.
+
+   * Create virtualenv for python.
+
+   * Run ``pip install -r transifex/requirements/microfrontend-common.txt``.
+
+   * Run the script that you try to create the jobs.
+
+     * Examples: `python transifex/pull.py https://github.com/edx/edx-platform.git edx/sustaining-team`. You might be required to setup environment variable ``GITHUB_ACCESS_TOKEN`` with your personal github account.
+
+       * For `push_translations`, you might also need transifex authentication. It is necessary to push first because you need transifex to be created before pull. If there are problems with transifex auth or priviledge, you should proceed to the next step and run push job once. 
```

Comment:
If we are to test our local changes, why do we need to run push first? Yes, push translations needs a certain permission on transifex, would that block others from doing push?
Also, if we are testing, can we assume some push has already been done for the the repo, unless its a brand new repo, in which case, what you say make more sense. Therefore, please spell out the specific scenarios to account for a larger audience